modules/ballerina-http/src/main/java/org/ballerinalang/net/http/HttpConnectionManager.java

```diff
@@ -172,9 +176,43 @@ public HttpClientConnector getHTTPHttpClientConnector(String scheme, BConnector
         BStruct options = (BStruct) bConnector.getRefField(0);
         int followRedirect = options.getBooleanField(0);
         Long maxRedirectCount = options.getIntField(0);
+        String trustStoreFile = options.getStringField(0);
```

Comment:
Is it possible to have meaningful constants for these numbers ?

Reply:
Done